src/axom/sidre/core/MapCollection.hpp

```diff
@@ -59,13 +59,13 @@
  *
  *          - // Return pointer to item with given name (nullptr if none).
  *
- *               TYPE* getItem(const std::string& name);
- *               TYPE const* getItem(const std::string& name) const ;
+ *               T* getItem(const std::string& name);
```

Comment:
Why this change from `TYPE` to `T` throughout?

Reply:
I modified it before extending the `ItemCollection` hierarchy. It seemed cleaner/more consistent with our `C++` coding conventions.